### PR TITLE
Preparation for Packaging

### PR DESCRIPTION
**Pull request by Sparkier**
**Describe your improvements**
Started packaging process by changing the project structure and defining the setup configuration.

**To Reproduce**
Follow the steps outlined in the README.md.

### CODE REVIEW

**Comment by Sparkier** [comment]
Hey @fuzhanrahmanian, I started this PR to make packaging of this project possible. I noticed that currently, the `alexnet.py` uses `luna/pretrained_models/data/meta_clsloc.mat`, which is not included in the project. Should we include this or can we maybe get rid of this requirement?

**Comment by Sparkier** [comment]
I removed unused code from the alexnet file, so that the `.mat` file is not needed anymore. However, in `googlenet.py`, we still have:
`googlenet.load_weights(os.path.join(os.path.dirname(__file__), 'googlenet_weights.h5'))`, and in `models.py` we have `    return alexnet.alex_net("alexnet_weights.h5")`, which are referring to static files. We should think about how to remove that in a future PR.

File: requirements.txt [hunk 1]
```diff
@@ -1,55 +1,2 @@
...
-tensorflow-estimator==2.4.0
-termcolor==1.1.0
-tf-slim==1.1.0
-toml==0.10.2
-typed-ast==1.4.2
-typing-extensions==3.7.4.3
-urllib3==1.26.2
-Werkzeug==1.0.1
-wrapt==1.12.1
-zipp==3.4.0
```
Comment (fuzhanrahmanian) [inline requirements.txt:55]: Why are we removing all the requirements?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+
+[options]
+packages = find:
+python_requires = >=3.6
+install_requires =
+    tensorflow
+    tf_slim
+    keras
```
Comment (fuzhanrahmanian) [inline setup.cfg:22]: Is numpy not needed? (image.py uses it)

**Review by fuzhanrahmanian — APPROVED** [review]
Except "NumPy" in the install requirements, the changes look good, thanks!

File: setup.cfg (hunk 2 above)
Comment (Sparkier) [reply setup.cfg:22]: I think numpy is a dependency of TensorFlow, which is why it still works I assume. However, if we use it directly we should add it. I will do that.

**Pull request merged**